### DISCUSSION TITLE
Getting notified when avatar is updated

### DISCUSSION THREAD

**Discussion by dadepo** [community/community#131113] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Is there a way to be notified via Webhook when the avatar of an organisation that has installed a GitHub app changes.

Looking through the documentation life https://docs.github.com/en/webhooks/webhook-events-and-payloads could not see support for such.

**Comment by rishibharadwajsai** [comment, marked as answer]
GitHub does not provide a webhook specifically for notifying changes to an organization's avatar when a GitHub app is installed. GitHub webhooks primarily cover events related to repositories, organizations, deployments, and other repository-specific activities.

However, you can periodically check the avatar URL of the organization using GitHub's REST API to detect changes. Here’s a basic approach you could take:

**Retrieve Organization Information**: Use the GitHub REST API to fetch information about the organization. You can do this using the following endpoint:
   ```
   GET /orgs/:org
   ```
   Replace `:org` with the actual name of the organization.
